docs/src/main/sphinx/installation/cli.rst

```diff
@@ -376,7 +376,13 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
-.. _cli-output-format:
+Auto suggestion
+---------------
+As you type commands in the CLI , you see offered a suggestion after the cursor
+based on history of commands already entered.
+
+Press the :kbd:→ key to accept the suggestion and replace the current command line buffer. Continue typing to reject the suggestion.
```

Comment:
Extremely minor: please put a line break after "command" so that line lengths are relatively uniform.